### PR TITLE
yeah boiii chat filter

### PR DESCRIPTION
**Pull request by Hour-of-the-Owl**
damage and bonds monitors

### CODE REVIEW

**Comment by HasKha** [comment]
Why did you change the style of the FAQs? I don't like the space between the question and the answer.

File: docs/pages/settings_folder.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ layout: default
 
 # Settings Folder
 
-**Work in progress**
+*Work in progress*
 
-*A section explaining how to open it*
+The Toolbox Settings Folder is found under `C:\Users\[Username]\AppData\Local`. You can open it by clicking the button at the top of the [Settings](settings) window.
```
Comment (HasKha) [inline docs/pages/settings_folder.md]: Should say `C:\Users\[Username]\AppData\Local\GWToolbox\`

We should also say how to open it without having toolbox opened, by typing `%LOCALAPPDATA%` in run/start menu/explorer path

**Comment by Hour-of-the-Owl** [comment]
I put the filepath... I thought that was enough xD

The FAQs just had a load of backslashes before, so I didn't really know what you wanted. I looked up how to fix that though, sooo are you happy now? =P

**Comment by HasKha** [comment]
yes, and the backslashes are there for a reason. Check https://haskha.github.io/GWToolboxpp/pages/faq . That is how it looks before your edits. By removing the backslashes, the answer to each question will be in a separate paragraph.

**Comment by Hour-of-the-Owl** [comment]
Okay well that's what it looks like to me on my version... not sure why it's different on yours, but never mind. I put the backslashes in, anyway.

Do you think I should put "How do I target a boo?!" on the main FAQ page or is there somewhere else it belongs?

**Comment by HasKha** [comment]
It's because you see it with GitHub markdown, the website is parsed by jekyll, and they have small differences. We talked about this :)

I don't know.

**Comment by Hour-of-the-Owl** [comment]
yeah I remember - it's just a nuisance trying to write in one language and predict what it'll look like in another xD

Is there anything else I need to do or can you accept this please? I want to make sure I didn't give the hotkeys section AIDS...

**Pull request merged**